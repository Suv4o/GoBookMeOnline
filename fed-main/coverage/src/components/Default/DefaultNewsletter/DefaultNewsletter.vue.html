
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for src/components/Default/DefaultNewsletter/DefaultNewsletter.vue</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../../index.html">All files</a> / <a href="index.html">src/components/Default/DefaultNewsletter</a> DefaultNewsletter.vue</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>81/81</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>81/81</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
        <template id="filterTemplate">
            <div class="quiet">
                Filter:
                <input oninput="onInput()" type="search" id="fileSearch">
            </div>
        </template>
    </div>
    <div class='status-line high'></div>
    <pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a>
<a name='L2'></a><a href='#L2'>2</a>
<a name='L3'></a><a href='#L3'>3</a>
<a name='L4'></a><a href='#L4'>4</a>
<a name='L5'></a><a href='#L5'>5</a>
<a name='L6'></a><a href='#L6'>6</a>
<a name='L7'></a><a href='#L7'>7</a>
<a name='L8'></a><a href='#L8'>8</a>
<a name='L9'></a><a href='#L9'>9</a>
<a name='L10'></a><a href='#L10'>10</a>
<a name='L11'></a><a href='#L11'>11</a>
<a name='L12'></a><a href='#L12'>12</a>
<a name='L13'></a><a href='#L13'>13</a>
<a name='L14'></a><a href='#L14'>14</a>
<a name='L15'></a><a href='#L15'>15</a>
<a name='L16'></a><a href='#L16'>16</a>
<a name='L17'></a><a href='#L17'>17</a>
<a name='L18'></a><a href='#L18'>18</a>
<a name='L19'></a><a href='#L19'>19</a>
<a name='L20'></a><a href='#L20'>20</a>
<a name='L21'></a><a href='#L21'>21</a>
<a name='L22'></a><a href='#L22'>22</a>
<a name='L23'></a><a href='#L23'>23</a>
<a name='L24'></a><a href='#L24'>24</a>
<a name='L25'></a><a href='#L25'>25</a>
<a name='L26'></a><a href='#L26'>26</a>
<a name='L27'></a><a href='#L27'>27</a>
<a name='L28'></a><a href='#L28'>28</a>
<a name='L29'></a><a href='#L29'>29</a>
<a name='L30'></a><a href='#L30'>30</a>
<a name='L31'></a><a href='#L31'>31</a>
<a name='L32'></a><a href='#L32'>32</a>
<a name='L33'></a><a href='#L33'>33</a>
<a name='L34'></a><a href='#L34'>34</a>
<a name='L35'></a><a href='#L35'>35</a>
<a name='L36'></a><a href='#L36'>36</a>
<a name='L37'></a><a href='#L37'>37</a>
<a name='L38'></a><a href='#L38'>38</a>
<a name='L39'></a><a href='#L39'>39</a>
<a name='L40'></a><a href='#L40'>40</a>
<a name='L41'></a><a href='#L41'>41</a>
<a name='L42'></a><a href='#L42'>42</a>
<a name='L43'></a><a href='#L43'>43</a>
<a name='L44'></a><a href='#L44'>44</a>
<a name='L45'></a><a href='#L45'>45</a>
<a name='L46'></a><a href='#L46'>46</a>
<a name='L47'></a><a href='#L47'>47</a>
<a name='L48'></a><a href='#L48'>48</a>
<a name='L49'></a><a href='#L49'>49</a>
<a name='L50'></a><a href='#L50'>50</a>
<a name='L51'></a><a href='#L51'>51</a>
<a name='L52'></a><a href='#L52'>52</a>
<a name='L53'></a><a href='#L53'>53</a>
<a name='L54'></a><a href='#L54'>54</a>
<a name='L55'></a><a href='#L55'>55</a>
<a name='L56'></a><a href='#L56'>56</a>
<a name='L57'></a><a href='#L57'>57</a>
<a name='L58'></a><a href='#L58'>58</a>
<a name='L59'></a><a href='#L59'>59</a>
<a name='L60'></a><a href='#L60'>60</a>
<a name='L61'></a><a href='#L61'>61</a>
<a name='L62'></a><a href='#L62'>62</a>
<a name='L63'></a><a href='#L63'>63</a>
<a name='L64'></a><a href='#L64'>64</a>
<a name='L65'></a><a href='#L65'>65</a>
<a name='L66'></a><a href='#L66'>66</a>
<a name='L67'></a><a href='#L67'>67</a>
<a name='L68'></a><a href='#L68'>68</a>
<a name='L69'></a><a href='#L69'>69</a>
<a name='L70'></a><a href='#L70'>70</a>
<a name='L71'></a><a href='#L71'>71</a>
<a name='L72'></a><a href='#L72'>72</a>
<a name='L73'></a><a href='#L73'>73</a>
<a name='L74'></a><a href='#L74'>74</a>
<a name='L75'></a><a href='#L75'>75</a>
<a name='L76'></a><a href='#L76'>76</a>
<a name='L77'></a><a href='#L77'>77</a>
<a name='L78'></a><a href='#L78'>78</a>
<a name='L79'></a><a href='#L79'>79</a>
<a name='L80'></a><a href='#L80'>80</a>
<a name='L81'></a><a href='#L81'>81</a>
<a name='L82'></a><a href='#L82'>82</a></td><td class="line-coverage quiet"><span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">&lt;script lang="ts"&gt;
export default {
  name: 'DefaultNewsletter',
}
&lt;/script&gt;
&nbsp;
&lt;template&gt;
  &lt;div class="py-16 sm:py-24"&gt;
    &lt;div class="relative sm:py-16"&gt;
      &lt;div aria-hidden="true" class="hidden sm:block"&gt;
        &lt;div class="absolute inset-y-0 left-0 w-1/2 bg-white rounded-r-3xl" /&gt;
        &lt;svg class="absolute top-8 left-1/2 -ml-3" width="404" height="392" fill="none" viewBox="0 0 404 392"&gt;
          &lt;defs&gt;
            &lt;pattern
              id="8228f071-bcee-4ec8-905a-2a059a2cc4fb"
              x="0"
              y="0"
              width="20"
              height="20"
              patternUnits="userSpaceOnUse"
            &gt;
              &lt;rect x="0" y="0" width="4" height="4" class="text-gray-200" fill="currentColor" /&gt;
            &lt;/pattern&gt;
          &lt;/defs&gt;
          &lt;rect width="404" height="392" fill="url(#8228f071-bcee-4ec8-905a-2a059a2cc4fb)" /&gt;
        &lt;/svg&gt;
      &lt;/div&gt;
      &lt;div class="mx-auto max-w-md px-4 sm:max-w-3xl sm:px-6 lg:max-w-7xl lg:px-8"&gt;
        &lt;div class="relative rounded-2xl px-6 py-10 bg-teal-600 overflow-hidden shadow-xl sm:px-12 sm:py-20"&gt;
          &lt;div aria-hidden="true" class="absolute inset-0 -mt-72 sm:-mt-32 md:mt-0"&gt;
            &lt;svg
              class="absolute inset-0 h-full w-full"
              preserveAspectRatio="xMidYMid slice"
              xmlns="http://www.w3.org/2000/svg"
              fill="none"
              viewBox="0 0 1463 360"
            &gt;
              &lt;path
                class="text-teal-500 text-opacity-40"
                fill="currentColor"
                d="M-82.673 72l1761.849 472.086-134.327 501.315-1761.85-472.086z"
              /&gt;
              &lt;path
                class="text-teal-700 text-opacity-40"
                fill="currentColor"
                d="M-217.088 544.086L1544.761 72l134.327 501.316-1761.849 472.086z"
              /&gt;
            &lt;/svg&gt;
          &lt;/div&gt;
          &lt;div class="relative"&gt;
            &lt;div class="sm:text-center"&gt;
              &lt;h2 class="text-3xl font-extrabold text-white tracking-tight sm:text-4xl"&gt;
                Want product news and updates?
              &lt;/h2&gt;
              &lt;p class="mt-6 mx-auto max-w-2xl text-lg text-teal-200"&gt;Sign up for our newsletter to stay up to date.&lt;/p&gt;
            &lt;/div&gt;
            &lt;form action="#" class="mt-12 sm:mx-auto sm:max-w-lg sm:flex"&gt;
              &lt;div class="min-w-0 flex-1"&gt;
                &lt;label for="cta-email" class="sr-only"&gt;Email address&lt;/label&gt;
                &lt;input
                  id="cta-email"
                  type="email"
                  class="block w-full border border-transparent rounded-md px-5 py-3 text-base text-gray-900 placeholder-gray-500 shadow-sm focus:outline-none focus:border-transparent focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-teal-600"
                  placeholder="Enter your email"
                /&gt;
              &lt;/div&gt;
              &lt;div class="mt-4 sm:mt-0 sm:ml-3"&gt;
                &lt;button
                  type="submit"
                  class="block w-full rounded-md border border-transparent px-5 py-3 bg-teal-500 text-base font-medium text-white shadow hover:bg-teal-400 focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-teal-600 sm:px-10"
                &gt;
                  Notify me
                &lt;/button&gt;
              &lt;/div&gt;
            &lt;/form&gt;
          &lt;/div&gt;
        &lt;/div&gt;
      &lt;/div&gt;
    &lt;/div&gt;
  &lt;/div&gt;
&lt;/template&gt;
&nbsp;</pre></td></tr></table></pre>

                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank" rel="noopener noreferrer">istanbul</a>
                at 2022-09-25T02:56:50.658Z
            </div>
        <script src="../../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../../sorter.js"></script>
        <script src="../../../../block-navigation.js"></script>
    </body>
</html>
    