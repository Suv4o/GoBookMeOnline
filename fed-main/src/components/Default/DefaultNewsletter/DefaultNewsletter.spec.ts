import { render, cleanup } from '@testing-library/vue'
import { describe, it, expect } from 'vitest'
import DefaultNewsletter from './DefaultNewsletter.vue'

describe('DefaultNewsletter', async () => {
  it('render component correctly', async () => {
    render(DefaultNewsletter)
    cleanup()
  })

  it('snap shot matches', async () => {
    const wrapper = render(DefaultNewsletter)
    expect(wrapper).toMatchSnapshot()
  })
})
