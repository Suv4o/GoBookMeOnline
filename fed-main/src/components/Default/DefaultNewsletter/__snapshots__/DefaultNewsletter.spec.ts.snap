// Vitest Snapshot v1

exports[`DefaultNewsletter > snap shot matches 1`] = `
{
  "baseElement": <body>
    <div>
      <div
        class="py-16 sm:py-24"
      >
        <div
          class="relative sm:py-16"
        >
          <div
            aria-hidden="true"
            class="hidden sm:block"
          >
            <div
              class="absolute inset-y-0 left-0 w-1/2 bg-white rounded-r-3xl"
            />
            <svg
              class="absolute top-8 left-1/2 -ml-3"
              fill="none"
              height="392"
              viewBox="0 0 404 392"
              width="404"
            >
              <defs>
                <pattern
                  height="20"
                  id="8228f071-bcee-4ec8-905a-2a059a2cc4fb"
                  patternUnits="userSpaceOnUse"
                  width="20"
                  x="0"
                  y="0"
                >
                  <rect
                    class="text-gray-200"
                    fill="currentColor"
                    height="4"
                    width="4"
                    x="0"
                    y="0"
                  />
                </pattern>
              </defs>
              <rect
                fill="url(#8228f071-bcee-4ec8-905a-2a059a2cc4fb)"
                height="392"
                width="404"
              />
            </svg>
          </div>
          <div
            class="mx-auto max-w-md px-4 sm:max-w-3xl sm:px-6 lg:max-w-7xl lg:px-8"
          >
            <div
              class="relative rounded-2xl px-6 py-10 bg-teal-600 overflow-hidden shadow-xl sm:px-12 sm:py-20"
            >
              <div
                aria-hidden="true"
                class="absolute inset-0 -mt-72 sm:-mt-32 md:mt-0"
              >
                <svg
                  class="absolute inset-0 h-full w-full"
                  fill="none"
                  preserveAspectRatio="xMidYMid slice"
                  viewBox="0 0 1463 360"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="text-teal-500 text-opacity-40"
                    d="M-82.673 72l1761.849 472.086-134.327 501.315-1761.85-472.086z"
                    fill="currentColor"
                  />
                  <path
                    class="text-teal-700 text-opacity-40"
                    d="M-217.088 544.086L1544.761 72l134.327 501.316-1761.849 472.086z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <div
                class="relative"
              >
                <div
                  class="sm:text-center"
                >
                  <h2
                    class="text-3xl font-extrabold text-white tracking-tight sm:text-4xl"
                  >
                     Want product news and updates? 
                  </h2>
                  <p
                    class="mt-6 mx-auto max-w-2xl text-lg text-teal-200"
                  >
                    Sign up for our newsletter to stay up to date.
                  </p>
                </div>
                <form
                  action="#"
                  class="mt-12 sm:mx-auto sm:max-w-lg sm:flex"
                >
                  <div
                    class="min-w-0 flex-1"
                  >
                    <label
                      class="sr-only"
                      for="cta-email"
                    >
                      Email address
                    </label>
                    <input
                      class="block w-full border border-transparent rounded-md px-5 py-3 text-base text-gray-900 placeholder-gray-500 shadow-sm focus:outline-none focus:border-transparent focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-teal-600"
                      id="cta-email"
                      placeholder="Enter your email"
                      type="email"
                    />
                  </div>
                  <div
                    class="mt-4 sm:mt-0 sm:ml-3"
                  >
                    <button
                      class="block w-full rounded-md border border-transparent px-5 py-3 bg-teal-500 text-base font-medium text-white shadow hover:bg-teal-400 focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-teal-600 sm:px-10"
                      type="submit"
                    >
                       Notify me 
                    </button>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="py-16 sm:py-24"
    >
      <div
        class="relative sm:py-16"
      >
        <div
          aria-hidden="true"
          class="hidden sm:block"
        >
          <div
            class="absolute inset-y-0 left-0 w-1/2 bg-white rounded-r-3xl"
          />
          <svg
            class="absolute top-8 left-1/2 -ml-3"
            fill="none"
            height="392"
            viewBox="0 0 404 392"
            width="404"
          >
            <defs>
              <pattern
                height="20"
                id="8228f071-bcee-4ec8-905a-2a059a2cc4fb"
                patternUnits="userSpaceOnUse"
                width="20"
                x="0"
                y="0"
              >
                <rect
                  class="text-gray-200"
                  fill="currentColor"
                  height="4"
                  width="4"
                  x="0"
                  y="0"
                />
              </pattern>
            </defs>
            <rect
              fill="url(#8228f071-bcee-4ec8-905a-2a059a2cc4fb)"
              height="392"
              width="404"
            />
          </svg>
        </div>
        <div
          class="mx-auto max-w-md px-4 sm:max-w-3xl sm:px-6 lg:max-w-7xl lg:px-8"
        >
          <div
            class="relative rounded-2xl px-6 py-10 bg-teal-600 overflow-hidden shadow-xl sm:px-12 sm:py-20"
          >
            <div
              aria-hidden="true"
              class="absolute inset-0 -mt-72 sm:-mt-32 md:mt-0"
            >
              <svg
                class="absolute inset-0 h-full w-full"
                fill="none"
                preserveAspectRatio="xMidYMid slice"
                viewBox="0 0 1463 360"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  class="text-teal-500 text-opacity-40"
                  d="M-82.673 72l1761.849 472.086-134.327 501.315-1761.85-472.086z"
                  fill="currentColor"
                />
                <path
                  class="text-teal-700 text-opacity-40"
                  d="M-217.088 544.086L1544.761 72l134.327 501.316-1761.849 472.086z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <div
              class="relative"
            >
              <div
                class="sm:text-center"
              >
                <h2
                  class="text-3xl font-extrabold text-white tracking-tight sm:text-4xl"
                >
                   Want product news and updates? 
                </h2>
                <p
                  class="mt-6 mx-auto max-w-2xl text-lg text-teal-200"
                >
                  Sign up for our newsletter to stay up to date.
                </p>
              </div>
              <form
                action="#"
                class="mt-12 sm:mx-auto sm:max-w-lg sm:flex"
              >
                <div
                  class="min-w-0 flex-1"
                >
                  <label
                    class="sr-only"
                    for="cta-email"
                  >
                    Email address
                  </label>
                  <input
                    class="block w-full border border-transparent rounded-md px-5 py-3 text-base text-gray-900 placeholder-gray-500 shadow-sm focus:outline-none focus:border-transparent focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-teal-600"
                    id="cta-email"
                    placeholder="Enter your email"
                    type="email"
                  />
                </div>
                <div
                  class="mt-4 sm:mt-0 sm:ml-3"
                >
                  <button
                    class="block w-full rounded-md border border-transparent px-5 py-3 bg-teal-500 text-base font-medium text-white shadow hover:bg-teal-400 focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-teal-600 sm:px-10"
                    type="submit"
                  >
                     Notify me 
                  </button>
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "emitted": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "html": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
